public/css/index.css

```diff
@@ -1,6 +1,6 @@
 .font {
     font-family: "Ubuntu", "Calibri";
```

Comment:
Please remove this font-family, we do not want to specify it but rather want browser to pick it automatically.

Reply:
Base font command not supported in new browsers.